README.md

```diff
@@ -10,9 +10,14 @@ Pandoc Reader is a [Pelican][] plugin that converts documents written in [Pandoc
 Requirements
 ------------
 
-This plugin requires a `pandoc` executable to be available on your `PATH`. We recommend using the latest version of Pandoc (2.11 and higher), as earlier versions are not supported.
+This plugin requires:
 
-To install this needed dependency, follow the [Pandoc installation instructions](https://pandoc.org/installing.html).
+* Python 3.7 or higher; and
+* Pandoc 2.11 or higher.
+
+To install Pandoc follow [these instructions](https://pandoc.org/installing.html).
```

Comment:
I suggest writing this as:
```
* Python 3.7 or higher; and
* Pandoc 2.11 or higher [[Pandoc installation instructions](https://pandoc.org/installing.html)].
```
… and then removing the sentence starting with "To install Pandoc follow…"